scripts/apps/search/controllers/SearchMenuController.ts

```diff
@@ -1,26 +1,61 @@
+import {get, isEqual} from 'lodash';
+import {ISavedSearch} from '../SavedSearch';
 
```

Comment:
What does the dropdown in the search page contain now? It used to only contain a provider e.g. a source a search to be executed on. Does the dropdown now contain both providers and shortcuts? Do shortcuts work with any provider? I would split them into groups so it's easy to know which ones are providers/shortcuts.